zwave_js_server/client.py

```diff
@@ -7,9 +7,10 @@
 from typing import Any, Dict, Optional, cast
 
 from aiohttp import ClientSession, ClientWebSocketResponse, WSMsgType, client_exceptions
-from packaging.version import parse as parse_version
+from awesomeversion import AwesomeVersion, AwesomeVersionException
```

Comment:
The exception is never raised in awesomeversion.
```suggestion
from awesomeversion import AwesomeVersion
```

Reply:
Yeah, it's coming in the next version that is being worked on .

Reply:
Next version is hitting a delay. Just going to remove this :)